master/internal/api_command.go

```diff
@@ -27,3 +40,52 @@ func (a *apiServer) KillCommand(
 	_ context.Context, req *apiv1.KillCommandRequest) (resp *apiv1.KillCommandResponse, err error) {
 	return resp, a.actorRequest(fmt.Sprintf("/commands/%s", req.CommandId), req, &resp)
 }
+
+func (a *apiServer) LaunchCommand(
```

Comment:
@stoksc any suggestions on how we should get better code reuse between shell and commands? they are so similar even golint is complaining. I explicitly wanted to keep them separate and don't share code between them.